### PR TITLE
Form

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team, Please review my code

live preview: https://infallible-mclean-1bb6f3.netlify.app/ 

Go on to the user icon to see the live preview for these two pages

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
name , alt is blank . code is good to merge .

File: login.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          <a href="#">Forgot your Password?</a></label
+        >
+      </div>
+
+      <div class="hero-btns">
+        <button class="btn">Login</button>
+      </div>
+      <br />
+      <div class="create">
+        <a href="signup.html">Create New Account ></a>
```
Comment (amirnathani9) [inline login.html:67]: here you can give some descriptive class name. Don't use common/single names for classes.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+  --border: 0.1rem solid rgba(0, 0, 0, 0.1);
+  --border-hover: 0.1rem solid var(--black);
+  --box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
+  --body-bg: #f8edeb;
+  --border-color: #9a9a9a;
+  --bg-color: #ffffff;
+  --font-color: #9a9a9a;
+  --btn-color: #3b82f6;
+  --default-color: #000000;
+  --pink: #e76f51;
```
Comment (amirnathani9) [inline style.css:17]: here is one suggestion, try to give a common name to a variable instead of color names

File: style.css [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+  color: var(--black);
+  border: var(--border);
+}
+
+.btn {
+  margin-top: 1rem;
+  display: inline-block;
+  padding: 0.9rem 3rem;
+  border-radius: 0.5rem;
+  color: #fff;
```
Comment (amirnathani9) [inline style.css]: use the color variable

File: style.css [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+  flex-flow: column wrap;
+  padding: 2rem 0;
+  margin: 1rem auto;
+}
+
+.feature .banner-img {
+  max-width: 100%;
+  height: auto;
+  border: 0.1rem solid rgba(0, 0, 0, 0.3);
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.1);
```
Comment (amirnathani9) [inline style.css]: use color variable or if you are using this color only here than give the comment

File: style.css [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+  .header .header-1 {
+    padding: 2rem;
+  }
+
+  section {
+    padding: 3rem 2rem;
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (amirnathani9) [inline style.css:354]: for media query also u can give the variable and use it. Decide the common breakpoints and define the mobile size, tablet size. This is only a suggestion.

**Comment by amirnathani9** [comment]
Your Ui is awesome and the code you have written is very neat and clean.
One suggestion I will give that use a more descriptive class name instead of single naming.

File: login.html (hunk 1 above)
Comment (sahiba0915) [reply login.html:67]: done

File: style.css (hunk 2 above)
Comment (sahiba0915) [reply style.css:17]: done

File: style.css (hunk 3 above)
Comment (sahiba0915) [reply style.css]: done

File: style.css (hunk 4 above)
Comment (sahiba0915) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (sahiba0915) [reply style.css:354]: done

**Pull request merged**